buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -499,6 +499,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def spark3_version = "3.1.2"
     def spotbugs_version = "4.0.6"
     def testcontainers_version = "1.16.3"
+    def testcontainers_version_clickhouse = "1.17.3"
```

Comment:
What is the reason to not just update the testcontainers version globally?

Reply:
@johnjcasey  Yes i can, just was not sure that it is OK i will do that

Reply:
@johnjcasey I have pushed the change testcontainers version